app/src/main/java/com/powerhungers/fomezero/feature/registration/domain/repository/RegistrationRepository.kt

```diff
@@ -0,0 +1,9 @@
+package com.powerhungers.fomezero.feature.registration.domain.repository
+
+import com.powerhungers.fomezero.feature.registration.domain.model.Register
+import kotlinx.coroutines.flow.Flow
+
+interface RegistrationRepository {
+    fun registration(name: String, userType: String,
+                     email: String, password: String): Flow<Register>
```

Comment:
```suggestion
    fun registration(name: String, userType: String, email: String, password: String): Flow<Register>
```
a identação desse metodo nao está correta e o nome register nao deixa claro o que ele faz

Reply:
assumo que ele registre o usuario entao sugiro que o nome possa ser registerUser()